libarchive/archive_write_set_format_pax.c

```diff
@@ -45,6 +45,8 @@ __FBSDID("$FreeBSD: head/lib/libarchive/archive_write_set_format_pax.c 201162 20
 #include "archive_write_private.h"
 #include "archive_write_set_format_private.h"
 
+#define USTAR_MAX_MTIME 0x1ffffffff
```

Comment:
I'm trying to remember the C rules for hex literals.  Does this need an `L` or `LL` suffix to ensure it gets handled correctly on 32-bit platforms?  Should it be cast to explicit `time_t`?  (I guess I should go dig through some references.)

Reply:
I dug through some C references and I think this works fine as-is.  Sorry for any confusion.